internal/rpmmd/repository.go

```diff
@@ -322,39 +325,31 @@ func runDNF(dnfJsonPath string, command string, arguments interface{}, result in
 		arguments,
 	}
 
-	cmd := exec.Command(dnfJsonPath)
```

Comment:
> This looks very interesting and I like where this is going! However I have a major issue with the current version.
> [...]
> Running `dnf-json` as a service probably makes sense, however personally I think that the generation of image test cases should be kept simple (executing a single command). This could mean that all tooling is modified to spin up `dnf-json` service in the background and stop it once it is not needed any more, or that `dnf-json` preserves the ability to be run as a command, not only as a service. I don't have a strong preference, but I think that this needs to be solved before the PR can be merged.

Thanks a lot! Yes you are right. I'm gonna change the tooling so that it uses the dameon version.